src/components/App/App.jsx

```diff
@@ -42,12 +43,15 @@ export default function App() {
 
   const sampleRecipeInfo = constRecipeInfo
 
+  const api_url = "http://localhost:3001/apirecipes/"
```

Comment:
nit: maybe `recipeApiUrl` to make this more clear when reading the code that uses it. Especially since you may add other APIs in the future.